src/machine/image-dbus.c

```diff
@@ -477,9 +478,11 @@ int image_node_enumerator(sd_bus *bus, const char *path, void *userdata, char **
         if (!images)
                 return -ENOMEM;
 
-        r = image_discover(IMAGE_MACHINE, images);
-        if (r < 0)
-                return r;
+        if (MANAGER_IS_SYSTEM(m)) {
+                r = image_discover(IMAGE_MACHINE, images);
+                if (r < 0)
+                        return r;
+        }
```

Comment:
we should probably add some dir for this too?

Reply:
This is something I avoided, because generally handling root-less images/containers is a rabbit hole. Although I wish systemd-nspawn would learn to work root-less somehow.

The new series introduce per-user machine&runtime paths. But it only works with root for now.

Is that acceptable enough for now?